packages/gafl-webapp-service/src/pages/contact/contact/__tests__/route.spec.js

```diff
@@ -181,14 +190,17 @@ describe('name > route', () => {
       expect(result.errorMessage).toBe(expected)
     })
 
-    it.each([
-      [true, '12M'],
-      [false, '8D'],
-      [false, '1D']
-    ])('result.twelveMonthLicence returns %s when licence length is %s', async (expected, licenceLength) => {
+    it.each`
+      desc                                                                               | licenceLength | junior   | expected
+      ${'twelveMonthNonJuniorLicence returns true when 12 month licence and not junior'} | ${'12M'}      | ${false} | ${true}
+      ${'twelveMonthNonJuniorLicence returns false when 8 day licence and not junior'}   | ${'8D'}       | ${false} | ${false}
+      ${'twelveMonthNonJuniorLicence returns false when 1 day licence and not junior'}   | ${'1D'}       | ${false} | ${false}
+      ${'twelveMonthNonJuniorLicence returns false when no licence length and junior'}   | ${undefined}  | ${true}  | ${false}
+    `('$desc', async ({ licenceLength, junior, expected }) => {
```

Comment:
again, a static parameterised string would be better than having description as a parameter, please